tests/test_refspec/test5.yml

```diff
@@ -6,4 +6,4 @@ repos:
 
   kas:
     url: https://github.com/siemens/kas.git
-    refspec: master@907816a5c4094b59a36aec12226e71c461c05b77
+    refspec: 907816a5c4094b59a36aec12226e71c461c05b77@master
```

Comment:
Reasoning?

Reply:
Then choose one option (I personally find the first one more natural, bu that's just one opinion) and fold that into the introducing patch. I would mirror the new version of this MR to the list for you.